src/shivani/Assignment_30/FindSecandMaxFromAraay.java

```diff
@@ -0,0 +1,29 @@
+package shivani.Assignment_30;
+/*WAP to find Second max from given array
+int[] arr = {10,33,43,55,97,11,3};
+ouput : 55*/ 
+public class FindSecandMaxFromAraay {
+	int max=0;
+	int secMax=0;
+	int findMaxArray(int[] arr) {
+		for(int i =0;i<arr.length;i++) {
```

Comment:
This can be handle with single loop
Take reference of below code
Shashank - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1560/files